locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_("")
```

Comment:
Not sure I see the connection between that code and this file.  I'm not sure what's autogenerated in this file, or which rake task runs that?